samples/packages/awscdk-typescript/src/construct/services.ts

```diff
@@ -1,14 +1,24 @@
 import { Vpc } from "aws-cdk-lib/aws-ec2";
 import { DockerImageAsset, NetworkMode } from "aws-cdk-lib/aws-ecr-assets";
-import { Cluster, ContainerImage, FargateService, FargateTaskDefinition, Protocol } from "aws-cdk-lib/aws-ecs";
+import {
+  Cluster,
+  ContainerImage,
+  FargateService,
+  FargateTaskDefinition,
+  Protocol,
+} from "aws-cdk-lib/aws-ecs";
 import { Construct } from "constructs";
-import { PackageInfraConfig, PackageServiceContainerEcsConfig, ServiceApplicationType } from "../package-config";
+import {
+  PackageInfraConfig,
+  PackageServiceContainerEcsConfig,
+  ServiceApplicationType,
+} from "../package-config";
 import * as path from "path";
 import { ApplicationLoadBalancedFargateService } from "aws-cdk-lib/aws-ecs-patterns";
 
 export interface ServicesProps {
-  config: PackageInfraConfig,
-  vpc: Vpc,
+  config: PackageInfraConfig;
+  vpc: Vpc;
```

Comment:
I've always used `,` in interfaces and `;` in classes for some reason

Reply:
Will change. 

Presume we can get the linter to catch these in the future. Will take a look.

Reply:
changed